Library/Formula/luabind.rb

```diff
@@ -58,6 +58,22 @@ def install
     end
     args << "--prefix=#{prefix}"
     system "bjam", *args
+
+    (lib/"pkgconfig/luabind.pc").write pc_file
+  end
+
+  def pc_file; <<-EOS.undent
```

Comment:
Don't mind it this way personally. It's consistent with our [Luas](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/lua.rb#L114-L135).